routes/users/[name]/index.ts

```diff
@@ -0,0 +1,13 @@
+import type { GithubUser } from "~types";
+
+export default eventHandler(async (event) => {
+  const name = getRouterParam(event, "name");
+  const user = await ghFetch(`users/${name}`);
+
+  return {
+    user: <GithubUser>{
+      id: user.id,
+      username: user.login,
```

Comment:
We can additionally expose `name`, `twitter_username` as `twitter`, and `avatar`